implementation/jwt-build/src/test/java/io/smallrye/jwt/build/JwtSignTest.java

```diff
@@ -386,6 +386,31 @@ void signWithShortRSAKeyAndRelaxedValidation() throws Exception {
         }
     }
 
+    @Test
+    void signWithoutAddingDefaultClaim() throws Exception {
+        KeyPair keyPair = KeyUtils.generateKeyPair(2048);
+
+        JwtBuildConfigSource configSource = getConfigSource();
+        configSource.setAddDefaultClaims(false);
+        try {
+            String jwt = Jwt.claims(Collections.singletonMap("customClaim", "custom-value"))
+                    .sign(keyPair.getPrivate());
+
+            JsonWebSignature jws = getVerifiedJws(jwt, keyPair.getPublic(), true);
+            JwtClaims claims = JwtClaims.parse(jws.getPayload());
+
+            assertEquals(1, claims.getClaimsMap().size());
```

Comment:
In all other tests where these 3 default claims are added, the expected number of claims is set to 4